kubectl-metalk8s/cmd/kubectl-metalk8s.go

```
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"flag"
+	"github.com/spf13/pflag"
+
+	"github.com/scality/metalk8s/kubectl-metalk8s/pkg/cmd"
+
+	"k8s.io/klog"
+)
+
+func main() {
+	flags := pflag.NewFlagSet("kubectl-metalk8s", pflag.ExitOnError)
+	pflag.CommandLine = flags
+
+	klog.InitFlags(nil)
+
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+
+	cmd.Execute()
```

Exceptions are ignored?

---

Exceptions handled by `Execute` function itself
```
// Execute adds all child commands to the root command and sets flags appropriately.
// This is called by main.main(). It only needs to happen once to the rootCmd.
func Execute() {
        if err := rootCmd.Execute(); err != nil {
                fmt.Println(err)
                os.Exit(1)
        }
}

```